debian/kolibri.scripts-common

```diff
@@ -284,6 +284,9 @@ kolibri_configure()
     # Remove because we might re-register as systemd during an upgrade
     update-rc.d -f kolibri remove
 
+    KERNEL_RASPBERRY=`uname -a|grep arm`
+    [ ! -z "$KERNEL_RASPBERRY" ] && echo "\n\n  NB! This operation take some minutes in this platform..."
```

Comment:
Apologies for not noticing this earlier, but this phrase seems to be missing a verb...?

Suggest changing from:

`This operation take some minutes in this platform...`

to

`This operation can take some time, please be patient!`

(since the display is already conditioned to appear _only_ on RPi, no need to add "on this platform")

cc @benjaoming @jredrejo